### ISSUE TITLE
[BUG] Cant's install firmware

### ISSUE THREAD

**Issue by d3lt4mik3** [CR6Community/Marlin#290]
### Did you test the latest `extui` code or prerelease?

Yes, and the problem still exists.

### Bug Description

When trying to install Community Firmware on CR6 SE with 1.1.0.3 board something goes wrong because screen red bar goes full and stays like this. Tried with several sd cards always respecting instructions for updating the firmware.
Updated the screen first and printer after but no luck. Screen goes everything ok but with printer i cant's install it.

### Version of CR6Community Firmware

6.1

### Printer model

Creality CR-6 SE

### Motherboard

Other (I will explain below)

### Display

Creality CR-6 stock touch screen

**Comment by ritchiedc** [comment]
Did you also flash the screen firmware and if so did you include the kernel upgrade files?

**Comment by d3lt4mik3** [comment]
Yes, in fact, i flashed the screen first. Copied the Dwin folder to a formated sd card with all files and was everything ok, but even so, i flashed it 3 times just to be sure that this was not the problem.
When i went back to the stock firmware on screen the build information of the firmware installed on printer was the stock one, so something is happening that prevents flashing with the community firmware, maybe my board, i don't know.

**Comment by Sebazzz** [comment]
You may need to retry several times. Creality stock boards are notorious for partially applying firmware updates. Make sure you've also applied the kernel update to the screen.

**Comment by d3lt4mik3** [comment]
I tried several times... 
Im not the only one with this problem.

https://damsteen.nl/blog/2021/03/27/creality-cr6-community-firmware-release-6-is-here

Honestly i dont know what can i do more. Its not my first printer.
Maybe someone can figure it out

**Comment by ritchiedc** [comment]
Did you copy the 3 files (files only, not the folder) in the kernel upgrade folder to the dwin_set folder? A photo of the blue screen when the screen flash is done would be helpful.

**Comment by d3lt4mik3** [comment]
Those files are by default in the folder of the screen firmware, and yes flash was successful like shown on instructions, but for sure, i made it 3 times.
The problem is on the board not the screen i think

**Comment by Sebazzz** [comment]
It would be interesting to see a picture of the screen PCB. From a good source I have understood the old DWIN displays that were delivered in 2020-now printers are no longer produced and Creality is now shipping new displays.

**Comment by d3lt4mik3** [comment]
I can provide this but not now cause its printing with the stock firmware. This printer was purchased 5 days ago. Was a promo on tomtop.com

**Comment by N1co93** [comment]
I also bought the CR6-SE on tomtop on the 09. September. I successfully flashed the community firmware on the printer with board `CR-ERA_V1.1.0.3`

Is there anything I can help with?

**Comment by d3lt4mik3** [comment]
> I also bought the CR6-SE on tomtop on the 09. September. I successfully flashed the community firmware on the printer with board `CR-ERA_V1.1.0.3`
> 
> Is there anything I can help with?

Well, i did everything as described on instructions. If you did something diferent and worked please tell me

**Comment by ritchiedc** [comment]
If you are installing 6.1 the kernel upgrade files are not in the dwin_set folder by default. They have to be copied there.

**Comment by d3lt4mik3** [comment]
https://imgur.com/a/r8TovEn

Here are the files on dwin folder

**Comment by d3lt4mik3** [comment]
https://imgur.com/a/r8TovEn

Here are the files on dwin folder

**Comment by d3lt4mik3** [comment]
https://imgur.com/a/r8TovEn

Here are the files on dwin folder

**Comment by d3lt4mik3** [comment]
Btw, problem is not the screen but the board itself... Many people on cr6 facebook group with same board reporting that cant update with 6.1, but 6.0 is ok. I have not tested with 6.0 version

**Comment by ritchiedc** [comment]
If those are the files in dwin_set and you did not copy the kernel files to it then you are not flashing v6.1. If you have the ERA board you need to download "CF6.1-Final-cr6-se-v4.5.3-mb-2021-06-04-16-53.zip". Is this the file you downloaded? Please be patient with me. I am trying to eliminate the most likely problem.

**Comment by d3lt4mik3** [comment]
> If those are the files in dwin_set and you did not copy the kernel files to it then you are not flashing v6.1. If you have the ERA board you need to download "CF6.1-Final-cr6-se-v4.5.3-mb-2021-06-04-16-53.zip". Is this the file you downloaded? Please be patient with me. I am trying to eliminate the most likely problem.

Yes, that file .zip. I did all steps like devs wrote on this github. Problem is on the board side, not the screen. The screen shows everything flashed ok like instructions. I think that it freezes on the full red bar at startup because of the stock firmware in printer. 
Problem is that no matter what, i cant install the 6.1 firmwarexxxx.bin on the printers board.

**Comment by ritchiedc** [comment]
Try a different sd card with a fresh format. Also make sure you rename the .bin file. The name has to be different from the previous filename flash attempt.

**Comment by d3lt4mik3** [comment]
Already did that... As i said, its not my first printer... I have a custom ender 3 V2 and i compile the firmware myself.

**Comment by d3lt4mik3** [comment]
Update for other users with same problem. I managed to flash the printer. Flashed with 6.0 and after with 6.1 and everything is running properly. Used same sd card as before but this way, worked.
Cheers and thanks to everyone who tried to help.

**Comment by jfpk** [comment]
I am having the same problem, on a brand new printer. The display firmware flash appears to finish correctly.  When I remove the microSD card and power-cycle the printer, the splash screen appears on the display, with a red progress bar that fills up.  It wont progress past this.  I have re-flashed numerous times.

I have an ERA 1.1.0.3 motherboard on the printer, and have attached an image of the display board.

I only have one microSD card available at the moment.  But I have ordered another one to see if a different card will help.  My existing card is 64GB SDXC card, however I partitioned to 8GB and formatted FAT32/4096.  I know the instructions state SDXC cards wont work, so perhaps that is the issue.  However the display appeared to read all files correctly during the flash process.   The new card I ordered is SDHC.

I will also try flashing to the 6.0 release first, and then 6.1, as it appears that solved the issue for the original poster.

![stuck_after_flash_CR61](https://user-images.githubusercontent.com/12667515/146303847-2522e751-b354-4891-8015-87ae152294fb.JPG)
![display_board](https://user-images.githubusercontent.com/12667515/146303855-1171482a-5c9a-4f5c-abc4-07932b43c07a.JPG)

**Comment by ritchiedc** [comment]
Jfpk, did you copy the kernel upgrade files (three of them) to the dwin_set folder before flashing the screen?

**Comment by jfpk** [comment]
Yes I did copy those 3 files into the DWN_SET folder.

**Comment by jfpk** [comment]
I have now tried it on a 2nd microSD card, with the same results.  Here is the display of the flash in progress, the file counts appear to match what is on the card, so I think its reading correctly.

![display_flash](https://user-images.githubusercontent.com/12667515/146649484-1cc34037-7a3c-4457-8fa3-537b46747df6.JPG)

**Comment by Sebazzz** [comment]
What happens if you tap the Creality logo or the progress bar area?

Met vriendelijke groet,
Sebastiaan Dammann
________________________________
Van: jfpk ***@***.***>
Verzonden: Saturday, December 18, 2021 6:02:20 PM
Aan: CR6Community/Marlin ***@***.***>
CC: Sebastiaan Dammann ***@***.***>; Comment ***@***.***>
Onderwerp: Re: [CR6Community/Marlin] [BUG] Cant's install firmware (Issue #290)


I have now tried it on a 2nd microSD card, with the same results. Here is the display of the flash in progress, the file counts appear to match what is on the card, so I think its reading correctly.

[display_flash]<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F12667515%2F146649484-1cc34037-7a3c-4457-8fa3-537b46747df6.JPG&data=04%7C01%7C%7Ce5e8217d85fd49f9b2b708d9c248279c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637754437432491488%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=tX4CWs%2FznfWsZ%2BTLN2bGFELVyF%2FHZ6W3vBFvJp0KswQ%3D&reserved=0>

**Comment by jfpk** [comment]
The screen is un-responsive to input.  I have tried tapping all over the screen, and it doesnt respond.

**Comment by Sebazzz** [comment]
Try re-applying the CR-6 kernel up date files

Met vriendelijke groet,
Sebastiaan Dammann
________________________________
Van: jfpk ***@***.***>
Verzonden: Saturday, December 18, 2021 6:20:28 PM
Aan: CR6Community/Marlin ***@***.***>
CC: Sebastiaan Dammann ***@***.***>; Comment ***@***.***>
Onderwerp: Re: [CR6Community/Marlin] [BUG] Cant's install firmware (Issue #290)


The screen is un-responsive to input. I have tried tapping all over the screen, and it doesnt respond.

**Comment by geek111** [comment]
Have same issue :D But now know why I'm wrong. Probably you wrong too :D 
If you want upgrade firmware on printer you must put sd card on printer not screen. You put sd card on screen only if you want upgrade firmware on screen :) and probably for this you this mistake :)

**Comment by RWBRITT** [comment]
I have a v4.22 board and no touch screen. I installed bltouch and got factory firmware to flash from SD card. I want to add a filament detector and some filament profiles. I can compile for the STM32F103RET6_creality board and it doesn't ever install completely. I can put factory bin file on card and it will load. STM32F103RET6_creality_maple is the other option I can compile for. 🤷🏻‍♂️

**Comment by Djcrew6** [comment]
I am having the same problem, on a brand new printer. The display firmware flash appears to finish correctly. When I remove the microSD card and power-cycle the printer, the splash screen appears on the display, with a red progress bar that fills up. It wont progress past this. I have re-flashed numerous times.
I have an ERA 1.1.0.3 motherboard on the printer.

My  card is 4GB SDHC card, and have moved the files as stated in the instuctions

@jfpk did our test to first upgrade to 6.0 and then to 6.1 worked?

Thankfull for any help or tips you have!

Br Dan

**Comment by Djcrew6** [comment]
Update on my case.
Cant take full credit, had a big help from Davids post: https://disqus.com/by/disqus_SAhO6FGvYx/
Did as he did and it works like a charm.
So you need to upgrade the motherbord first, and then the screen.
But I notice one thing, you can´t use any SD card under 8 gig when doing the update on the motherboard 1.1.0.3, the motherbord wont notice it and then it wont upgrade. Tried alot of sizes under 8 gig with no sucsess. when I did it with a 8 gig, it workt like it shoud do.
Br Dan

**Comment by jfpk** [comment]
@Djcrew6 Thanks for the info.  I had given up on the upgrade,  I will try flashing motherboard first.
